docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
```

Comment:
You should update your README.md to mention running the app in Docker and what the correct command is.